bin/reformat_design.py

```diff
@@ -70,8 +70,13 @@ def reformat_design(DesignFileIn,DesignFileOut):
                 print "{}: Replicate id not an integer!\nLine: '{}'".format(ERROR_STR,line.strip())
                 sys.exit(1)
 
-            ## CHECK FASTQ FILES EXIST PER SAMPLE
             for fastq in fastQFiles:
+                ## CHECK FASTQ FILE EXTENSION
+                if fastq[-9:] != '.fastq.gz':
```

Comment:
What about `.fq.gz` files?

Reply:
I guess I insisted on `fastq.gz` as an extension because I rename all the files in the pipeline to use the names in the design file. This is currently hard-coded to use `fastq.gz` as an extension. It probably should be updated but I havent got round to playing around with it yet
https://github.com/nf-core/atacseq/blob/eaf8dba1d96186ca3c1c672ee553afdf911d1004/main.nf#L505-L513

Reply:
Actually...it may not be a problem to use `fq.gz`. Would require some testing though.

Reply:
It's just using `$reads` though - it's only the resulting filename that has `.fastq.gz` hardcoded right? So it should work with any input filename... (but I think that it's good to ensure that people don't try to use bam files or whatever)

Reply:
Yeah. I realised that when I looked back at the code. :+1: